test/Altinn.Notifications.Tests/Altinn.Notifications.Tests.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Common.AccessToken" Version="3.0.3" />
+    <PackageReference Include="Altinn.Common.AccessToken" Version="4.1.0" />
```

Comment:
This reference should be unnecessary.

Reply:
A package downgrade is introduced if we don't have this explicitly. Will keep for now.